File: frontend/cinema/src/app/services/actor/actor-dto.ts
export class ActorDto {

    id: any;
    name: string;
    age: number;
    movies = [];

    constructor(id, name, age, movies) {
        this.id = id;
        this.name = name;
        this.age = age;
        this.movies = movies;
    }
}

File: business/src/main/java/de/qaware/cinema/business/dto/ActorDto.java
package de.qaware.cinema.business.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class ActorDto {

    private Long id;
    private String name;
    private int age;
    private List<String> movies;
}
